draft-irtf-nwcrg-coding-and-congestion-04.xml

```diff
@@ -327,22 +327,93 @@
+			</section>
+
+			<section anchor="subsec:cc-recov-inter-below" title="FEC below the transport">
+				<t>The congestion control may not know what is going on in the network underneath and whether a coding scheme is introduced or not. The congestion control may behave as if no coding sheme is introduced. The only way for a coding channel to indicate that symbols have been recovered is to exploit existing signaling. An example would be a client to indicate that a packet has been recovered by using ECN signals of TCP.</t>
+			</section>
+		</section>
+
+		<section anchor="subsec:cc-nc-interaction" title="Interactions between congestion control and coding rates">
+			<t>This section discuss to what extent interaction between the congestion control and the coding rates are possible.</t>
+			<section title="FEC above the transport">
+				<t>The coding rate applied at the application mainly depends on the available capacity given by the congestion control underneath. Adapting the coding rate to the minimum required data rate of the application may help in reducing packet losses and improve quality of experience.</t>
+			</section>
+
+			<section title="FEC within the transport">
+				<t>In this case, there is an important flexibility in the trade-off between reducing goodput, helping recovering from transmission and congestion losses, being fair in respect with non-coded flows. Such as explained in <xref target="subsec:cc-recov-inter-in"></xref>, the client may be able to indicate the server the amount of packets that have been received or recovered. The server may exploit this information to tune the coding ratio. As one example of flexibility of this case, coupling increasing transmission rate with increasing or decreasing coding rate could be envisionned. A server may use increasing coding rate as a probe to more safely increase transmission rate.</t>
+			</section>
+
+			<section title="FEC below the transport">
+				<t>The coding rate in this case can be tuned depending on the amount of recovered symbols and the rate of which data are transmitted by the server. The coding scheme is not aware of the congestion control implementation making it hard for the coding scheme to apply the relevant coding rate.</t>
+			</section>
+		</section>
+
+		<section anchor="subsec:cc-useless-interaction" title="On the useless repaired symbols">
+			<t>There are cases where useless repaired symbols are transmitted and impact on the load on the network and may reduce the goodput of the flow without concrete gains.</t>
+			<section title="FEC above the transport">
+				<t>In this case, the discussion depends on application needs. The only case where adding useless repaired symbols does not result in reduced goodput is when the application needs a limited amount of goodput (e.g. VoIP traffic). In this case, the useless repaired symbols would only impact on the amount of data generated in the network.</t>
```

Comment:
```suggestion
				<t>In this case, the discussion depends on application needs. The only case where adding useless repaired symbols does not result in reduced goodput is when the application needs a limited amount of goodput (e.g. VoIP traffic). In this case, the useless repaired symbols would only have an impact on the amount of data generated in the network.</t>
```

Reply:
Done in commit 

commit b16e6519992ec31093ac529b4040f9585278227e

Resolve if OK